server/cluster/metadata/cluster_metadata_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2023 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package metadata_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/CeresDB/ceresmeta/server/cluster/metadata"
+	"github.com/CeresDB/ceresmeta/server/coordinator/procedure/test"
+	"github.com/CeresDB/ceresmeta/server/storage"
+	"github.com/stretchr/testify/require"
+)
+
+func TestClusterMetadata(t *testing.T) {
```

Comment:
Can you try to separate the tests?

Reply:
There are dependencies between the unit tests here and there, which cannot be directly divided into multiple.